### DISCUSSION TITLE
Module development

### DISCUSSION THREAD

**Discussion by SGr33n** [crater-invoice-inc/crater#935] [General]
Hi!
I'd like to create a new module for Crater but I cannot found any documentation. I just found "development" on the official webpage but it's just about installation. Do you have some starting point to begin the development?
Thanks!

**Comment by IvanBernatovic** [comment]
Bumping this up as I'm also interested in official documentation related to modules.

**Comment by michelmelo** [comment]
when do we have the documentation? It's been talked about for months and nothing, and no platform updates.
